metadata.go

```diff
@@ -16,12 +16,13 @@ const (
 )
 
 type BuildMetadata struct {
-	BOM        []BOMEntry       `toml:"bom" json:"bom"`
-	Buildpacks []GroupBuildpack `toml:"buildpacks" json:"buildpacks"`
-	Labels     []Label          `toml:"labels" json:"-"`
-	Launcher   LauncherMetadata `toml:"-" json:"launcher"`
-	Processes  []launch.Process `toml:"processes" json:"processes"`
-	Slices     []layers.Slice   `toml:"slices" json:"-"`
+	BOM                         []BOMEntry       `toml:"bom" json:"bom"`
+	Buildpacks                  []GroupBuildpack `toml:"buildpacks" json:"buildpacks"`
+	Labels                      []Label          `toml:"labels" json:"-"`
+	Launcher                    LauncherMetadata `toml:"-" json:"launcher"`
+	Processes                   []launch.Process `toml:"processes" json:"processes"`
+	Slices                      []layers.Slice   `toml:"slices" json:"-"`
+	BuildpackDefaultProcessType string           `toml:"buildpack-default-process-type, omitempty" json:"buildpack-default-process-type"`
```

Comment:
The `json` may need an omitempty as well

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/520/commits/7b24df17bb72f57cfba08d780795cbaba5aaddca.